core/builtins/parser/message.py

```
@@ -648,6 +648,8 @@ async def _execute_module_command(msg: "Bot.MessageSession", module, command_fir
                                                    module=command_first_word,
                                                    prefix=msg.session_info.prefixes[0]))
             return
+        except Exception as e:
+            raise e
```

Ruff (0.14.1)

Use `raise` without specifying exception name

Remove exception name

(TRY201)